marimo/_output/md.py

```diff
@@ -55,6 +55,8 @@ def _md(text: str, apply_markdown_class: bool = True) -> Html:
             "toc",
             # Footnotes
             "footnotes",
+            # Admonitions
+            "admonition",
```

Comment:
Oh sick, I thought the parser was FE markdown. Adding the extension and styles looks awesome.
No markdown integration needed, since it's just pure markdown as is.

Just out of curiosity, what does adding `pymdownx.details` look like?
https://facelessuser.github.io/pymdown-extensions/extensions/details/
Since pymdownx is already a dep